content/en/users/cloud-compute/ec3/_index.md

```diff
@@ -8,9 +8,20 @@ description: >
 
 [EC3](http://servproject.i3m.upv.es/ec3/) (Elastic Cloud Compute Cluster) is a
 tool to create elastic virtual clusters on Infrastructure as a Service
-(IaaS) providers. It supports the provisioning of clusters running TORQUE,
-SLURM, SGE, HTCondor, Mesos, Nomad, Kubernetes and others that will be
+(IaaS) providers, either public (such as [Amazon Web Services](https://aws.amazon.com/), 
+[Google Cloud](http://cloud.google.com/) or [Microsoft Azure](http://azure.microsoft.com/)) 
+or on-premises (such as [OpenNebula](http://www.opennebula.org/) and 
+[OpenStack](http://www.openstack.org/)).
+It supports the provisioning of clusters running
+[TORQUE](http://www.adaptivecomputing.com/products/open-source/torque),
+[SLURM](http://slurm.schedmd.com/), [HTCondor](https://research.cs.wisc.edu/htcondor/), 
+[Mesos](http://mesos.apache.org/), [Nomad](https://www.nomadproject.io/), 
+[Kubernetes](https://kubernetes.io/) and others that will be
 automatically resized to fit the load (e.g. number of jobs at the batch system).
 
-This section of the documentation will guide you on how to use EC3 with EGI Cloud
-providers using the OpenStack API and OpenID Connect for deploying your clusters.
+The following section of the documentation will guide you on how to:
+
+- Request access to EC3,
+- Deploy simple EC3 elastic cluster on top of the IaaS providers of the EGI Cloud,
+  either using the web interface or the command-line interface,
```

Comment:
do we have the web interface part? Maybe we should add that before

Reply:
That's the plan. Add a new section as reported in this doc: https://docs.google.com/document/d/14YZ2IQDU1p2gE0ofEfk_mJKUHFqnbJG7qv17mT-ki_c/edit#

Reply:
Then let's go in the right order, I would add those docs as part of this PR and we merge everything together

Reply:
Ok, fine with me.

Reply:
So if this PR was made with the online editor, I'm not sure how possible/easy it's to add other files to this PR.
May have to do checkout of the PR locally to a branch and push it, but I didn't test and document this...

Reply:
I did this once for another repo, as long as you keep committing changes to the same branch, things will be added here.

Reply:
So in fact it's easy with [GitHub CLI](https://docs.egi.eu/about/git/#starting-with-the-github-cli):
```sh
gh pr checkout 126
vim ...
git add ...
git commit ... -m xxx
git push
```

And it should just work!